docs/source/usage_guides/distributed_inference.md

```diff
@@ -216,13 +216,10 @@ with torch.no_grad():
     output = model(*args)
 ```
 
-When finished, all the data will be on the last GPU, which you can use the [`PartialState`] to find and extract:
+When finished all the data will be on the CPU on each process for convenience:
```

Comment:
we need to update this and let the user know that they can change this behavior using `send_output_to_cpu`.